.rubocop_todo.yml

```diff
@@ -195,62 +217,46 @@ Lint/UnusedBlockArgument:
     - 'test/dummy/app/controllers/custom/registrations_controller.rb'
     - 'test/dummy/config/initializers/omniauth.rb'
 
-# Offense count: 4
+# Offense count: 3
 # Cop supports --auto-correct.
 # Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
 Lint/UnusedMethodArgument:
   Exclude:
     - 'app/controllers/devise_token_auth/omniauth_callbacks_controller.rb'
     - 'app/controllers/devise_token_auth/unlocks_controller.rb'
-    - 'lib/devise_token_auth/engine.rb'
     - 'lib/generators/devise_token_auth/install_generator.rb'
 
-# Offense count: 3
+# Offense count: 2
 Lint/UriEscapeUnescape:
   Exclude:
     - 'test/controllers/devise_token_auth/omniauth_callbacks_controller_test.rb'
-    - 'test/controllers/devise_token_auth/registrations_controller_test.rb'
     - 'test/controllers/overrides/confirmations_controller_test.rb'
 
-# Offense count: 35
+# Offense count: 31
 Metrics/AbcSize:
   Max: 76
 
 # Offense count: 1
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
-  Exclude:
-    - 'app/models/devise_token_auth/concerns/user.rb'
+  Max: 42
 
-# Offense count: 15
+# Offense count: 14
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
   Max: 698
 
-# Offense count: 14
+# Offense count: 12
 Metrics/CyclomaticComplexity:
   Max: 17
 
-# Offense count: 92
+# Offense count: 74
 # Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
 # URISchemes: http, https
 Metrics/LineLength:
-  Exclude:
-    - 'app/controllers/devise_token_auth/application_controller.rb'
-    - 'app/controllers/devise_token_auth/concerns/set_user_by_token.rb'
-    - 'app/controllers/devise_token_auth/omniauth_callbacks_controller.rb'
-    - 'app/controllers/devise_token_auth/passwords_controller.rb'
-    - 'app/controllers/devise_token_auth/registrations_controller.rb'
-    - 'app/controllers/devise_token_auth/sessions_controller.rb'
-    - 'app/controllers/devise_token_auth/unlocks_controller.rb'
-    - 'app/models/devise_token_auth/concerns/user.rb'
-    - 'app/models/devise_token_auth/concerns/user_omniauth_callbacks.rb'
-    - 'app/validators/email_validator.rb'
-    - 'lib/devise_token_auth/engine.rb'
-    - 'lib/devise_token_auth/rails/routes.rb'
-    - 'lib/generators/devise_token_auth/install_generator.rb'
+  Max: 189
```

Comment:
why 189?

Reply:
The `--auto-gen-config` option sets this...  It picks whatever line-length necessary so that everything is ignored. Hence, I set it explicitly back to the default of `Max: 80` in the `.rubocop.yml` file.  I then manually exclude each file with a violation, instead of extending the max length.

Reply:
I just realized that this is a todo file, I commented this also because #1125 sets 80. Will you remove this file in the future?